src/components/Navbar/index.jsx

```diff
@@ -107,44 +146,70 @@ function Navbar({ t }) {
           {/* ----------- Languages ----------- */}
           <div className='md:flex items-end cursor-pointer hidden'>
             <div
-              className='flex items-end'
-              onClick={() => setLanguages(!languages)}
+              className='flex gap-1 hover:border-purple border-2 p-1 rounded-3xl'
+              onClick={toggleLanguage}
             >
-              <TfiWorld className='text-2xl text-gray-700' />
-              <TbArrowBadgeDown className='text-xl text-violet-700' />
-            </div>
-            <div className='relative'>
-              <div
-                className={`absolute left-[-70px] rtl:left-[-10px] top-5 bg-white capitalize ${
-                  languages ? `${styles.show}` : "hidden"
-                }`}
-              >
-                <div className='my-2 px-5 py-2 hover:bg-gray-200 transition-all duration-500'>
-                  <Link href='' locale='en'>
-                    English
-                  </Link>
-                </div>
-                <div className='my-2 px-5 py-2 hover:bg-gray-200 transition-all duration-500'>
-                  <Link href='' locale='ar'>
-                    العربية
-                  </Link>
-                </div>
-              </div>
+              <TfiWorld className='text-3xl text-gray-700' />
+              <span className='text-gray-700'>{activeLanguage}</span>
             </div>
           </div>
           {/* ----------- Buttons ----------- */}
           <div className='flex  justify-between items-center gap-5 flex-col md:flex-row '>
-            <div className='flex gap-3 capitalize flex-col md:flex-row'>
-              <Link href='/signin'>
-                <div className='bg-purple-light py-2 px-5 text-white rounded-3xl text-sm hover:bg-violet-800 transition-all cursor-pointer'>
-                  {t("sign-in")}
+            <div className='flex gap-4 capitalize flex-col md:flex-row'>
+              {user ? (
+                // Display user icon if signed in
+                <div className='flex items-center'>
+                  <div className='block md:relative text-center md:border-2 hover:border-purple rounded-full'>
```

Comment:
When you add a new image, it turns into an oval shape and not circular
Also, we need to change the image after adding a new image

Reply:
Ok